quarkus-workshop-super-heroes/super-heroes/rest-fights/src/main/java/io/quarkus/workshop/superheroes/fight/FightResource.java

```diff
@@ -1,22 +1,21 @@
 package io.quarkus.workshop.superheroes.fight;
 
+import jakarta.ws.rs.*;
```

Comment:
Not a fan of this kind of change personally, but it's minor :)

Reply:
Editorconfig is not very useful for formatting, IME, but it's good for this kind of thing: https://github.com/editorconfig/editorconfig/issues/338. We have a .editorconfig, but it's not got a rule for the wildcard limit. Let's add one, and then we never have to have this conversation again. :)

Reply:
👍🏼